README.md

```diff
@@ -197,7 +175,7 @@ Your working browser.
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](https://github.com/najibullahjafari/).
+Feel free to check the [issues page](https://github.com/najibullahjafari/math-magicians).
 
```

Comment:
- [ ] Great work so far but it would be nice if you replace the link with the link shown below which is the correct link to your issues page 👍

`https://github.com/najibullahjafari/math-magicians/issues`